docs/tokens/data-credit.mdx

```diff
@@ -128,25 +128,25 @@ import OperationCostEstimator from '@site/src/theme/DcCostEstimator'
 
 In addition to network data transfer, DCs are also utilized for certain network-related fees.
 
-| Action            | Fee                                             |
-| ----------------- | ----------------------------------------------- |
-| Device Onboarding | $40                                             |
-| Assert Location   | $10                                             |
-| OUI               | $100                                            |
-| DevAddr           | $100 ($800)<sup>(</sup>[^1]<sup>)</sup>         |
+| Action            | Fee                                     |
+| ----------------- | --------------------------------------- |
+| Device Onboarding | $40                                     |
+| Assert Location   | $10                                     |
+| OUI               | $100                                    |
+| DevAddr           | $100 ($800)<sup>(</sup>[^1]<sup>)</sup> |
 
 [^1]: DevAddrs must be purchased in slabs of 8.
 
 ## Data Credits and Mobile {#dc-and-mobile}
 
-In much the same design as the IoT network, the cellular network leverages DCs as the sole mechanism
+In much the same design as the IoT network, the Mobile network leverages DCs as the sole mechanism
 of billing and fee management. However, as a separate protocol, different pricing mechanics apply.
-While the IoT network charges data at a rate of $0.00001 per 24-byte message, the Mobile network
-charges data at a rate of **$0.50 per 1 gigabyte** (50,000 DC).
+While the Helium IoT network charges data at a rate of $0.00001 per 24-byte message, the Mobile
```

Comment:
The IoT network has already been introduced so we don't need to prefix Helium here.